.github/workflows/test_swiftui_install_script.yml

```diff
@@ -0,0 +1,29 @@
+name: Test SwiftUI Install Script
+
+on:
+  push:
+    branches: [ feature/**, bug/**, chore/** ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  Test:
+    name: Test
+    runs-on: macOS-12
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
```

Comment:
I think we should revise this since it's redundant fetching with all commits and tags as [document](https://github.com/actions/checkout/blob/96f53100ba2a5449eb71d2e6604bbcd94b9449b5/README.md?plain=1#L7) mentioned